elpis/transformer/elan.py

```diff
@@ -140,7 +140,7 @@ def update_ui(file_paths: List[Path], ui):
     return ui
 
 @elan.import_files('eaf')
-def import_eaf_file(eaf_paths, context, add_annotation, tmp_dir):
+def import_eaf_file(eaf_paths, context, reset_annotations, add_annotation, tmp_dir):
```

Comment:
Needs types, this file is convoluted enough already.

Reply:
Also why are there two?

Reply:
Two birds in the hand is better than one bird in the bush?? I removed one and added correct details for the things I understand.